### PR TITLE
Add new env called suppliers-sftp-bucket

### PR DESCRIPTION
**Pull request by tmckernan**
Add a new ENV variables for the supplier SFTP
 - suppliers-sftp-bucket

### CODE REVIEW

**Comment by adrianmilne** [comment]
Hi @tmckernan  - is this suppliers SFTP bucket one that we provision as part of our scripts?

**Comment by tmckernan** [comment]
@adrianmilne No that is `product-import` S3 bucket.  This is the S3 bucket which TechOps will set up and inside this bucket will be each suppliers own S3 bucket using AWS transfer.  Allowing a suppliers to upload files using SFTP.  The files which the different suppliers upload to these S3 buckets will get copy into the `product-import` bucket.

**Comment by adrianmilne** [comment]
@tmckernan - that's great thanks - just checking we didn't already know the name of that bucket in the code 👍

File: terraform/modules/services/spree/ecs.tf [hunk 1]
```diff
@@ -131,6 +131,7 @@ data "template_file" "app_client" {
     app_domain                 = var.app_domain
     papertrail_hostname        = var.papertrail_hostname
     papertrail_remote_port     = var.papertrail_remote_port
+    suppliers_sftp_bucket      = var.suppliers_sftp_bucket
```
Comment (adrianmilne) [inline terraform/modules/services/spree/ecs.tf:111]: Does the new variable also need adding to the `spree.json.tpl` file?

**Review by adrianmilne — APPROVED** [review]
👍  applied to SBX1

**Pull request merged**